backend/compact-connect/lambdas/provider-data-v1/data_model/schema/license.py

```diff
@@ -0,0 +1,70 @@
+# pylint: disable=invalid-name
```

Comment:
The schema classes all use naming conventions from the JSONSchema (camelCase), rather than from PEP8 style. Because basically the whole module is intentionally ignoring PEP8, I'm disabling that naming rule for the module, rather than disabling at every class attribute in-line.